### PR TITLE
Final Tweaks to Intro Map SVG

### PR DESCRIPTION
**Pull request by ellenbechtel**
Before making a pull request
----------------------------
First . . .
- [ ] Clean the code the way Vue likes it - run 'npm run lint --fix'        
- [ ] Make sure all tests run

Then check for accessibly compliance
- [ ] Run WAVE plugin 508 compliance tool

Then run Browserstack; check that application works on . . .
- [x] Chrome
- [ ] Safari
- [ ] Edge
- [x] Firefox
- [ ] Samsung Internet
- [ ] Internet Explorer 11 (not supported, but still needs at least a working user redirect page)

Finally . . .
- [ ] Update the changelog appropriately

Final Tweaks to Map
-----------
Brief description of changes. Reference the JIRA ticket if appropriate

Keep em coming with the nitpicks!  I did everything you mentioned except the following:

- New York tracking: I repositioned New York's label so that it was more centered and tighter tracking, so that it would appear in cropped mobile versions
- There is a light weight to this font (Source Sans Pro)!  The lightest is 200, regular is 400, and on up
- Made the glow less opaque, but didn't fundamentally change the rendering.  What do you think of this?
- What do you think of the cropping system I came up with for mobile? Once the screen size is below 600px, it crops off evenly on each side and repositions and a few key text elements (specifically, the scale bars and the Atlantic Ocean label).  I think cropping the edges feels fine for the most part, except that we lose state labels like "Maryland" and have to squish some things in.  

^ And alternate system to do this would be to NOT crop anything, but scale the map **DOWN** with screen size and scale the font sizes **UP** with screen size, plus maybe remove some labels altogether on mobile screens. I like that less, but certainly doable if you don't want to crop off the edges. 

-----------
If no ticket is referenced, describe the changes made. Note anything that you want the reviewers to know while
reviewing your pull request

After making a pull request
---------------------------
- [ ] If appropriate, put the link to the PR in the ticket
- [ ] Assign someone to review unless the change is trivial

### CODE REVIEW

**Comment by hcorson-dosch-usgs** [comment]
Nice, Ellen! This is looking _really_ good.

Sorry for the delay - I wanted to get some changes I was working on committed before I pulled down your branch. Here are my comments - I think the only piece of what is mentioned below that would be good to fine tune before moving on to another sprint task is the mobile cropping. The rest can wait for later tweaks, so long as we put them on a to-do list somehwere.

> New York tracking: I repositioned New York's label so that it was more centered and tighter tracking, so that it would appear in cropped mobile versions

Ah, that makes sense. I think that's definitely a good move if we take the cropping approach for mobile

> There is a light weight to this font (Source Sans Pro)! The lightest is 200, regular is 400, and on up

Sweet! I think that works great

> Made the glow less opaque, but didn't fundamentally change the rendering. What do you think of this?

I think it's definitely better, but is maybe too opaque now. I'd try something in between, maybe 70-80% opacity? I think I had it down at 50% opacity before, which we agreed was too low

> What do you think of the cropping system I came up with for mobile? Once the screen size is below 600px, it crops off evenly on each side and repositions and a few key text elements (specifically, the scale bars and the Atlantic Ocean label). I think cropping the edges feels fine for the most part, except that we lose state labels like "Maryland" and have to squish some things in.

Hmmm. This is tricky b/c I do like that cropping allows the actual watershed (and therefore the city and river labels within the watershed) to be bigger. However, if I toggle the mobile screen size in Firefox, the extent of the crop seems to vary (not sure if this is reflective of how it would actually display on mobile, though). I think if we were going to crop, we would need to drop all labels that aren't fully within the crop, to be sure that they aren't accidentally cut off on some mobile screens. That would probably mean having to drop labels for the Atlantic, Chesapeake Bay, Maryland, Pennsylvania, and Harrisburg. Also, with the crop, the map image seems to be too tall for some mobile screen sizes (e.g. iPhone 6/7/8 on firefox), so you can't see the whole watershed at once. It would be nice if we could fit at least the watershed itself into a single view on every device, if not the whole map. 

Now that I've written this all out and thought it over, I think cropping is the way to go, especially since I could see the alternate option of scaling the font sizes with different screen sizes introducing a lot of complexity. And I think having the watershed bigger due to the cropped view is a big win.


A few other little things I noticed while looking at the map:
* It looks like some of the city labels are too close to the city markers - notably Binghamton, Scranton, New York, and Washington.
* The city markers for the cities within the DRB appear to be beneath the city labels, so they are getting blocked by the mask around the city labels
* And a few more nitpicky things:
  * I think the color of the scale bar text is a bit too bright
  * I think the city labels for cities outside of the watershed should be a bit darker, so that they drop a bit in the visual hierarchy
  * And I think the state labels could also be a tiny bit darker, for the same reason
  * Right now the coastal buffers aren't quite showing. I think maybe my request to lighten the buffers yesterday was actually a bad one. So if we could go back to the color they were that would be great

File: src/components/ImportanceIcons.vue [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+        grid-area: desc-2; 
+    }
+
+    #desc-3 {
+        grid-area: desc-3; 
+    }
+
+    #desc-4 {
+        grid-area: desc-4; 
+    }
```
Comment (cnell-usgs) [inline src/components/ImportanceIcons.vue:205]: interesting. I didn't know about `grid-area` but seems like a good approach

**Review by cnell-usgs — APPROVED** [review]

**Comment by ellenbechtel** [comment]
@hcorson-dosch How about this for mobile views: I'll make a class that removes certain labels when we go to mobile, so that they disappear entirely rather than half-appear-half-get-cropped when the screen size is smaller. I'll work on that this morning and make another PR with that change and the others you mentioned for one final review before moving on to something else.

**Comment by hcorson-dosch-usgs** [comment]
Sounds great, @ellenbechtel !

**Pull request merged**